com.unity.shadergraph/Editor/Drawing/PreviewManager.cs

```diff
@@ -295,14 +295,19 @@ static void ForeachConnectedNode(AbstractMaterialNode node, PropagationDirection
                 }
             }
 
-            // Custom Interpolator Blocks have implied connections to their Custom Interpolator Nodes
+            // Custom Interpolator Blocks have implied connections to their Custom Interpolator Nodes...
             if (dir == PropagationDirection.Downstream && node is BlockNode bnode && bnode.isCustomBlock)
             {
                 foreach (var cin in CustomInterpolatorUtils.GetCustomBlockNodeDependents(bnode))
                 {
                     action(cin);
                 }
             }
+            // ... Just as custom Interpolator Nodes have implied connections to their custom interpolator blocks
+            if (dir == PropagationDirection.Upstream && node is CustomInterpolatorNode ciNode)
+            {
+                action(ciNode.e_targetBlockNode);
+            }
```

Comment:
#3652 solves SRP batching by gathering properties ahead of time. This case wasn't necessary for the previous execution flow, but was leaving it out was still an oversight on my part.